src/GameObjects/Bullet.py

```diff
@@ -0,0 +1,16 @@
+import pygame
+
+from Interface.GameObjectInterface import GameObject
+
+class Bullet(GameObject):
+    def __init__(self, x, y):
+        self.x = x
+        self.y = y
+        # change the hitbox size depending on the Image size for player
+        self.hitbox = (self.x, self.y, 20, 20)
+
+    def drawCharacter(self, canvas):
+        pass
+
+    def propagate(self, step):
+        pass
```

Comment:
A file should always end in a new-line.
That is why it is showing an error here..
Please add a new line

Reply:
Please change the word Player in comments..
It should be "Bullet"